### ISSUE TITLE
Dendron's wiki navigation is cluttered and overwhelming. 

### ISSUE THREAD

**Issue by LiminalCrab** [dendronhq/dendron#805]
## Please Select if your Request is Either something new or an Enhancement ##

- [X] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applys to. (Multiple selections are Possible) ##

- [ ] Onboard. Initial vault setup and import/export pods 
- [ ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [X] Misc (Choose this if your not sure)

## Problem ##

Navigating the wiki for relevant information is rather difficult, there is a lot of clutter in the navbar and information is scattered everywhere. It might be a put off for new comers looking to quickly jump into Dendron. I think it might be a good idea to clean it up.

## Solution ##

The root categories on the wiki should be reduced. I would recommend consolidating everything into less than five descriptive categories. Here's a quick idea. 

### Documentation
All information related to actually using Dendron's feature set.

1. Installation
2. Getting Started
3. Features 
4. ~~Topic~~ --> API? Topic isn't very descriptive, in fact it's something I'd personally skip over at first glance.
5. Workflow
6. Cheatsheet
7. FAQ

### Development
All information related to the development of Dendron.

1. Releases
2. RFC
3. Reference
4. Contribute
5. Case Studies

### Miscellaneous 
Information someone might seek, but the overwhelming majority of users didn't come to the wiki for.

1. Principles
2. Testimonials
3. Community
4. Jobs
5. Dendron Alternatives
6. Related Projects

## Additional context ##

I didn't go through the hassle of including a markdown hierarchical structure for these, but I believe most of them should stay as is, that being said I think there should be a strong focus on consolidating the information on the wiki. It actually might be a good idea to add a table of content just beneath the wiki's root directory "Dendron" above "Getting started" with links / a small description of the information contained in that page. 

Not to crutch on a disability here, but as someone with ADHD it's hard finding the signal in the noise.

**Comment by micharris42** [comment]
Going to +1 this. I realize the focus at the moment is improving/expanding Dendron's feature set, and documentation is probably less of a priority at this time, but if I may be brutally honest, one of the first things that came to mind upon my first visit to the wiki was: "I am going to need to set aside a few hours in my calendar to read through all this."

I think consolidation is definitely the way to go. For an example, typing "Lookup" into the search bar (one of Dendron's premier features and one that it is important to differentiate from VS Code's built in Quick Open) brings up *five* different pages titled Lookup:

1. dendron.dev.design.lookup
2. pro.dendron-plugin.internal.lookup
3. pro.dendron-plugin.internal
4. dendron.topic.lookup
5. pro.dendron-engine.internal.lookup

Followed by a whopping *80 pages* in which Lookup is mentioned. I counted. 

I know the Dendron site proper got a fresh coat of paint not that long ago. Perhaps some of that could be spared to tidy up the wiki?

**Comment by flammehawk** [comment]
I would probably recommend renaming  `topic` to `features` since that would make more sense in my eyes. 

Secondly, maybe we should additionally think about splitting the Developer Doku out into `wiki-dev.dendron.so`, so that the on site search results get better.

**Comment by kpathakota** [comment]
Yeah, this is good feedback. I think we'll need to tackle this asap to avoid overwhelming new folks too. Thanks for pointing this out @LiminalCrab 

+1 on splitting the developer docs out too @flammehawk

**Comment by SR--** [comment]
+1 I would suggest adding some simple step-by-step recipes and DOs and DON'Ts. One can easily do a lot of breaking by modifying file names, links, frontmatter, moving, deleting, all that with typical OS operations. It is scary to new users and should be clarified upfront, with recovery methods suggested.
